google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -294,12 +326,59 @@ def website_404= website_404
         end
 
         ##
+        # Indicates that a client accessing the bucket or a file it contains
+        # must assume the transit costs related to the access. The requester
+        # must pass the `user_project` option to {Project#bucket} to indicate
+        # the project to which the access costs should be billed.
+        #
+        # This feature is currently available only to whitelisted projects.
+        #
+        # @return [Boolean, nil] Returns `true` if requester pays is enabled for
+        #   the bucket.
+        #
+        def requester_pays
+          @gapi.billing.requester_pays if @gapi.billing
+        end
+        alias_method :requester_pays?, :requester_pays
+
+        ##
+        # Enables requester pays for the bucket. If enabled, a client accessing
+        # the bucket or a file it contains must assume the transit costs related
+        # to the access. The requester must pass the `user_project` option to
+        # {Project#bucket} to indicate the project to which the access costs
+        # should be billed.
+        #
+        # This feature is currently available only to whitelisted projects.
+        #
+        # @param [Boolean] new_requester_pays When set to `true`, requester pays
+        #   is enabled for the bucket.
+        #
+        # @example
```

Comment:
Add description for this example for it to be consistent with other example. Suggestion: Enable requester pays for a bucket.